M2/cmake/check-libraries.cmake

```diff
@@ -178,7 +178,6 @@ endif()
 ###############################################################################
 ## TODO: Do we still want these libraries?
 #   fplll	Lattice algorithms using floating-point arithmetic	(uses gmp and mpfr)
-#   linbox	Exact computational linear algebra	(needs fflas and givaro)
```

Comment:
My understanding was that Mike was still interested in experimenting with Linbox, even though it isn't actively in use.

Reply:
@mikestillman - What's the status of using linbox?

Reply:
I did have the plan to experiment with linbox, but I haven't done so recently.